kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/JmxClusterUtil.java

```diff
@@ -20,6 +21,7 @@
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.pool2.KeyedObjectPool;
+import org.jetbrains.annotations.Nullable;
```

Comment:
I'm not sure that we introduced this dependency.

Reply:
Indeed I don't see the dependency (jetbrains-annotations) being explicitly included anywhere but there are already multiple usages of these annotations found across the project. Do you want me to change them to javax annotations or let's keep em?

Reply:
Actually there's no difference at all between them.
If we intend to get rid of jetbrains' annotations in favor of javax annotations we need to somehow enforce usage of javax annotations, otherwise people will keep messing with them.
Considering all of them work atm and we don't wanna invent a way to enforce usage of specific ones, maybe just let it be?